addons/web_editor/static/src/js/base.js

```diff
@@ -1,8 +1,6 @@
 odoo.define('web_editor.base', function (require) {
 'use strict';
 
-// TODO this should be re-removed as soon as possible.
```

Comment:
I'd let that `TODO` because this entire module should be re-removed at some point. It was restored by the revert but could be useless by using other mechanism. In fact would it still be required by anything after your 13.0 fix is merged ?